### ISSUE TITLE
Unsafe array's length manipulation

### ISSUE THREAD

**Issue by bearcanrun** [Superalgos/ALGOToken#15]
This is most likely a non-issue. The warning is about directly changing the length of arrays (e.g. `.length++` instead of `.push()`).

Warning states: _The length of the dynamic array is changed directly. In this case, the appearance of gigantic arrays is possible and it can lead to a storage overlap attack (collisions with other data in storage)._
Source: [SmartCheck Analysis](https://tool.smartdec.net/scan/e3c7fbd47ddb46478e9ec131a222a519)

**Code:**

```
# L:55, array is reset to specific length of 1
_miners.length = 1; 

# L:66, array length is reduced by 1
_miners.length--; 

// Note: Starting with Solidity compiler 0.5.0 
// .pop() becomes available

```

**Comment by pmmax** [comment]
Fixed in #21

**Comment by bearcanrun** [comment]
Documentation updated.
